src/pybind/mgr/dashboard/frontend/src/app/app-routing.module.ts

```diff
@@ -335,10 +332,10 @@ const routes: Routes = [
     component: BlankLayoutComponent,
     children: [
       // Single Sign-On (SSO)
-      { path: 'sso/404', component: SsoNotFoundComponent },
+      { path: 'sso/404', component: ErrorComponent },
       // System
-      { path: '403', component: ForbiddenComponent },
-      { path: '404', component: NotFoundComponent },
+      { path: '403', component: ErrorComponent },
```

Comment:
Probably we should simply have a generic `/error` page accepting either params passed by url `/error/:message`, or via `params`, or the new [`state`](https://netbasal.com/set-state-object-when-navigating-in-angular-7-2-b87c5b977bb).